src/test/scala/ai/privado/languageEngine/java/passes/config/PropertiesFilePassTest.scala

```diff
@@ -122,24 +126,71 @@ class GetPropertyTests extends PropertiesFilePassTestBase {
   }
 }
 
+// Unit test to check if property is added in the cpg using XML files.
+class XMLPropertyTests extends PropertiesFilePassTestBase(".xml") {
```

Comment:
I see only one use case added. I think there would be more combination of use cases we need to add

Reply:
Will add more test case coverage.